src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+    private int supply;
+    private int buy;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        if (file.exists()) {
+            try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
```

Comment:
Reading is also a separate operation and should be in a separate private method. The best solution is to have 3 private methods and 3 lines of code in getStatistic() method

Reply:
Extracted